Projects/Home/Sources/HomeState.swift

```diff
@@ -0,0 +1,110 @@
+import Apollo
+import Flow
+import Presentation
+import hCore
+import hGraphQL
+
+public struct HomeState: StateProtocol {
+	var memberContractState: MemberContractState = .loading
+
+	public init() {}
+}
+
+public enum HomeAction: ActionProtocol {
+	case openFreeTextChat
+	case fetchMemberState
+	case openMovingFlow
+	case openClaims
+	case connectPayments
+	case setMemberContractState(MemberContractState)
```

Comment:
Associated values in codable must be named:

```suggestion
	case setMemberContractState(state: MemberContractState)
```